t/130-release-all-tasks-on-start.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env tarantool
```

Comment:
Please, add the executable bit for the file.

Reply:
Done.